package.json

```diff
@@ -13,7 +13,8 @@
     "test-ercmocks": "env-cmd -e development mocha --timeout 0 --bail test/e2e/ercmocks.test.mjs",
     "lint": "eslint . --ext js,mjs,jsx && find-unused-exports",
     "prepare": "husky install",
-    "doc:build:sdk": "jsdoc -c jsdoc.json cli/lib/nf3.mjs"
+    "doc:build:sdk": "jsdoc -c jsdoc.json cli/lib/nf3.mjs",
+    "block-zero-test": "env-cmd -e development mocha --timeout 0 --bail test/block-zero-challenge.mjs"
```

Comment:
It's great there is a test case for this - but I'm not sure if we want to be adding these specific kinds of test to the repo. Providing instructions to apply to existing test (if they're simple enough) should be sufficient.

E.g. for this one, would it not be sufficient to comment out parts of `neg-http` to recreate `block-zero-challenge`

Reply:
Yes . That should be sufficient . But @ChaitanyaKonda suggested to add a separate test file to make it easier for reviewers who are reviewing without context. Once the issue receive enough approvals , I will push a new PR without the issue specific test file.

Reply:
@IlyasRidhuan is right. My earlier suggestion was for a bug fix that we weren't going to replicate with an easy change to the test. If we can do a quick change to test file by adding something as simple as `.skip` to some tests, then we should do it. This will eliminate the additional step of a temporary test file and having to remove it. 

Another suggestion was to have a test file as `gist` that reviewers could copy + paste and test with, but you can't push to gist.  Could you check that again please?